packages-ts/starknet-gauntlet/src/wallet/index.ts

```diff
@@ -1,34 +1,25 @@
-import { ec, KeyPair, Signer } from 'starknet'
+import { SignerInterface } from 'starknet'
+import { Wallet as DefaultWallet } from './defaultWallet'
+import { Wallet as LedgerWallet } from './ledgerWallet'
+
 export interface IWallet<W> {
   wallet: W
-  sign: (message: any) => any
   getPublicKey: () => Promise<string>
 }
 
-export interface IStarknetWallet extends IWallet<Signer> {
+export interface IStarknetWallet extends IWallet<SignerInterface> {
   getAccountPublicKey: () => string
```

Comment:
The public key is one thing, an account address is another:

```suggestion
  getAccountAddress: () => string
```